app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::API
+  def param_finder
```

Comment:
is this needed in every single controller? if not, could we find another place to put this? we don't need every controller to know about this method if we can avoid it